pkg/api/requests/auth.go

```diff
@@ -10,3 +10,9 @@ type AuthTokenGet struct {
 type AuthTokenSwap struct {
 	TenantParam
 }
+
+// AuthTokenGet is the structure to represent the request data for get auth token endpoint.
+type AuthCustomToken struct {
+	Name      string `json:"name"`
+	ExpiresAt int    `json:"expired"`
```

Comment:
I believe the JSON name could be "expires_at". Additionally, these two fields could also have a `validate` tag with a required field.

Reply:
adjusted to expires_at and include tags to validate